packages/content/src/attributes/types/strings/AbstractEMailAddress.ts

```diff
@@ -4,19 +4,22 @@ import { AbstractString } from "../AbstractString";
 
 export abstract class AbstractEMailAddress extends AbstractString {
     // from https://html.spec.whatwg.org/multipage/input.html#valid-e-mail-address
+    private static readonly regExp = new RegExp(
+        /^[A-Za-z0-9!#$%&'*+/=?^_`{|}~-]+(\.[A-Za-z0-9!#$%&'*+/=?^_`{|}~-]+)*@([A-Za-z0-9ÄäÖöÜüß]([A-Za-z0-9ÄäÖöÜüß-]{0,61}[A-Za-z0-9ÄäÖöÜüß])?\.)+[A-Za-z0-9ÄäÖöÜüß][A-Za-z0-9ÄäÖöÜüß-]{0,61}[A-Za-z0-9ÄäÖöÜüß]$/
```

Comment:
The part left of the '@' is forbids non-ascii characters. Add them like you did in the domain part of the email address. Don't forget to add a test.

Reply:
It was said that we should allow non-ascii characters in the domains (you can see it in the ticket) that's why I didn't add them there. So should I?